web/gds-admin-ui/src/pages/app/details/TrixoQuestionnaire/index.js

```diff
@@ -31,36 +22,38 @@ function TrixoForm({ data }) {
+                                            <li data-testid='other-juridiction' key={index}>{countryCodeEmoji(validateIsoCode(juridiction?.country))} {isoCountries[juridiction?.country]} regulated by {juridiction?.regulator_name}</li>
                                         )
                                     })
                                 }
-                            </li>
+                            </ul>
                         </ul>
-                        <p className='lh-lg'>Organization <span className='fw-bold'>{getMustComplyRegulations(data?.must_comply_travel_rule)}</span> comply with the application of the Travel Rule standards in the jurisdiction(s) where it is licensed/approved/registered.</p>
+                        <p data-testid="must-comply-travel-rule" className='lh-lg'>Organization <span className='fw-bold'>{getMustComplyRegulations(data?.must_comply_travel_rule)}</span> comply with the application of the Travel Rule standards in the jurisdiction(s) where it is licensed/approved/registered.</p>
                     </Col>
                     <Col xs={12} md={6}>
                         <h5 className='text-black'>Applicable Regulations</h5>
                         <hr className='mt-1' />
                         <ul>
                             {
-                                data?.applicable_regulations.map(regulation => <li key={regulation}>{regulation}</li>)
+                                data?.applicable_regulations?.map(regulation => <li key={regulation}>{regulation}</li>)
                             }
                         </ul>
-                        <p><span className='fw-bold'>Minimum Compliance Threshold:</span> {intlFormatter({ currency: data?.compliance_threshold_currency }).format(data?.compliance_threshold)} {data?.compliance_threshold_currency}</p>
+                        <p><span className='fw-bold'>Minimum Compliance Threshold:</span> <span data-testid="compliance_threshold_currency">{intlFormatter({ currency: data?.compliance_threshold_currency }).format(data?.compliance_threshold)}</span> {data?.compliance_threshold_currency}</p>
                     </Col>
                     <Col xs={12} md={6}>
                         <h5 className='text-black'>CDD & Travel Rule Policies</h5>
                         <hr className='mt-1' />
-                        <p className='lh-lg'>
+                        <p className='lh-lg' data-testid="has-required-regulatory-program">
                             Organization <span className='fw-bold'>{hasRequiredRegulatoryProgram[data?.has_required_regulatory_program]}</span> programme that sets minimum AML, CFT, KYC/CDD and Sanctions standards per the requirements of the jurisdiction(s) regulatory regimes where it is licensed/approved/registered.
```

Comment:
```suggestion
                            Organization <span className='fw-bold'>{hasRequiredRegulatoryProgram[data?.has_required_regulatory_program]}</span> meet minimum AML, CFT, KYC/CDD and Sanctions standards per the requirements of the jurisdiction(s) regulatory regimes where it is licensed/approved/registered.
```

Reply:
`Organization has a partially implemented programme that sets minimum AML` What will happen if we have a sentence like this ?

Reply:
This wording makes sense to me, can we change `programme` to `program` to be consistent with the other US english spelling? This actually brings up a really good point that we haven't officially made a decision of which spelling to prefer (US english vs. British english), but for right now let's stick to US english spelling of words to be consistent with the rest of the UI.